src/Roslyn.Diagnostics.Analyzers/Core/RoslynDiagnosticsAnalyzersResources.resx

```diff
@@ -255,4 +255,13 @@
   <data name="FixNumberedComments" xml:space="preserve">
     <value>Fix numbered comments</value>
   </data>
+  <data name="DefaultableTypeShouldHaveDefaultableFieldsDescription" xml:space="preserve">
+    <value>Defaultable types should have defaultable fields</value>
+  </data>
+  <data name="DefaultableTypeShouldHaveDefaultableFieldsMessage" xml:space="preserve">
+    <value>Defaultable types should have defaultable fields</value>
```

Comment:
We always add more context to the diagnostic message so it is useful from CI.
```suggestion
    <value>Defaultable type '{0}' should not have a non-defaultable field '{1}'</value>
```

Reply:
Done